.gitignore

```
@@ -36,3 +36,6 @@ wandb/
 checkpoints/
 results/
 code_snapshots/
+
+# Runtime env
+runtime_env.yaml
```

what about:
```suggestion
*runtime_env.yaml
```
and providing a `default_runtime_env.yml` with the contents:
```yaml
env_vars:
    RAY_OVERRIDE_JOB_RUNTIME_ENV: "1" # This is required to avoid errors when job runtime_env conflicts with runtime_env set in driver
```

---

Done